products/datacatalog/terraform.yaml

```diff
@@ -84,6 +84,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "basic_tag_template"
         vars:
           tag_template_id: "my_template"
+          force_delete: "false"
```

Comment:
this is for the docs, correct?  do we want to put `false` in there yet before it's supported by the API?  I don't have a strong opinion either way, just wanted to check thoughts.

Reply:
Yup, same rationale here as the bigtable one where I prefer to recommend the safer option for users and make them explicitly have to switch over to the one that lets them delete the resource when they decide to do so.